torch/onnx/symbolic_helper.py

```diff
@@ -1598,11 +1580,14 @@ def quantize_helper(
         )
 
     assert scale is not None
-    if scale.type().scalarType() != "Float":
+    if _type_utils.JitScalarType.from_value(scale) != _type_utils.JitScalarType.FLOAT:
         scale = g.op("Cast", scale, to_i=_C_onnx.TensorProtoDataType.FLOAT)
 
     assert zero_point is not None
-    if zero_point.type().scalarType() not in ("Byte", "Char"):
+    if _type_utils.JitScalarType.from_value(zero_point) not in {
```

Comment:
ditto

Reply:
same as above

Reply:
discussed offline. refactroing symbolics to make sure intended type fallback is done